supyr_struct/field_types.py

```diff
@@ -736,8 +745,9 @@ def __call__(self, name, *desc_entries, **desc):
 
         desc[TYPE] = self
 
-        '''Remove '0  #' from this line to enable adding descriptor
-        entries to the descriptor rather than overwriting old ones.'''
+        # TODO: This comment suggests editing the library directly. This is bad.
+        # Remove '0  #' from this line to enable adding descriptor
+        # entries to the descriptor rather than overwriting old ones.
         i = 0  # desc.get(ENTRIES, 0)
```

Comment:
This was a comment left to myself as a reminder. I wasn't sure if I should allow that behavior or not. It can be removed.